src/style.css

```diff
@@ -1,3 +1,75 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  background-color: #fff;
+}
+
+@media (min-width: 1024px) and (max-width: 1200px) {
+  .header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 50px;
+  }
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-content: stretch;
+  flex-direction: row;
+  /* margin-right: 50px; */
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__link {
+  /* margin-right: 20px; */
```

Comment:
```suggestion

```

Reply:
variables for repeating colors changed, comments deleted